apps/projects/app/components/Content/Issues.js

```diff
@@ -545,7 +568,9 @@ class Issues extends React.PureComponent {
         onError={console.error}
       >
         {({ data, loading, error, refetch }) => {
-          if (data && data.node0) {
+          if ((data && data.node0) ||
+              (this.state.downloadedIssues != [] &&
+               this.state.downloadedRepos != {})) {
```

Comment:
Yes, we super do. They get populated with all the issues. Here I'm checking that the data was loaded from the API. Otherwise, display the `loading...` text from down below. But wait, we don't want to display that text if we already have some previously downloaded issues, do we? Hence the inelegant check. Probably downloadedIssues was enough but hey, you can't be too careful, right?

Reply:
> They get populated with all the issues.

Sorry, where? Somehow I'm missing where that happens.

Reply:
https://github.com/AutarkLabs/planning-suite/pull/1057/files/2b52b85557ea97153552d2666b6e176051cad38f#diff-1e61190978ae534ae80fe1f071b7e785R474

Reply:
Isn't that line updating the local variable, which is only set within the scope of `flattenIssues`? And then returning it to the caller of `flattenIssues`? Where does the global/state-level `downloadedIssues` get updated? I'm expecting a `setState` call somewhere. Same for `downloadedRepos`.

Reply:
https://github.com/AutarkLabs/planning-suite/pull/1057/files/2b52b85557ea97153552d2666b6e176051cad38f#diff-1e61190978ae534ae80fe1f071b7e785R498

Reply:
I don't think that line is working anymore

Reply:
That link no longer works

Reply:
Basically, the caller of flattenIssues takes the new value and then passes it to another function that looks for new issues and then updates the state. I don't remember the name of this other function and right now I can't look at the code, but it's the same function where you tracked the bug that you discovered before making the review. Remember?

Reply:
I don't think that function (the `render` function) calls `setState` with new data, either (which is good!). I'm also not seeing anything that directly (and ill-advisedly) updates `state.downloadedIssues`, using something like `this.state.downloadedIssues = ['new', 'array']`

Reply:
I was on my phone and ended up commenting on the main thread, I don't know if you noticed. Anyways, the place where `setState` is called is [this line](https://github.com/AutarkLabs/planning-suite/pull/1057/files#diff-1e61190978ae534ae80fe1f071b7e785R420) inside the `showMoreIssues` function.

Reply:
Oh, interesting. I didn't expect it to live there. This makes me want to organize all the logic in this file, but that's outside the scope of the current task 😄